_locales/en/messages.json

```diff
@@ -242,14 +242,21 @@
   "extensionHasUpdated": {
     "message": "Scratch Addons has updated to v$1"
   },
-  "extensionUpdateInfo1_v1_32": {
-    "message": "New addon: \"move sprite to front layer\", which lets you Shift+Click a sprite within the sprite pane to move it to the front (top layer) of the stage. Go to $1 to enable it."
+  "extensionUpdateInfo1_v1_33": {
+    "message": "New addon: \"highlight project creator in comments\", which adds a badge next to the project creator in project comments sections. Go to $1 to enable it."
   },
   "scratchAddonsSettings": {
     "message": "Scratch Addons settings"
   },
-  "extensionUpdateInfo2_v1_32": {
-    "message": "Other new addons added this update are \"faster project loading\", \"follow topics after posting\", \"necropost highlighter\", and \"remember collapsed forum categories\"."
+  "extensionUpdateInfo2_v1_33": {
+    "message": "Multiple addons have been adjusted in preparation for the upcoming Scratch changes. This update also brings performance improvements, increased storage for addon settings, and more."
```

Comment:
```suggestion
    "message": "Multiple addons have been adjusted in preparation for the upcoming Scratch changes. This update also brings performance and stability improvements."
```

Reply:
Nice!

Reply:
I know understand why the update messages in the App Store back in 2013 were so vague. Miss my iPod Touch running iOS 6.